app/main.py

```diff
@@ -1,8 +1,49 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(
+        self,
+        comfort_class: int,
+        clean_mark: int,
+        brand: str
+    ):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(
+        self,
+        distance_from_city_center: int,
+        clean_power: int,
+        average_rating: float,
+        count_of_ratings: int
+    ):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_mark = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, car: Car):
```

Comment:
```suggestion
    def serve_cars(self, cars: Car):
```
I believe, this is a list of the cars